### ISSUE TITLE
Tagging images is deprecated

### ISSUE THREAD

**Issue by michaelsauter** [opendevstack/ods-jenkins-shared-library#67]
See:
```
[Pipeline] stage
[Pipeline] { (Deploy to Openshift)
[Pipeline] openshiftTag
NOTE: steps like this one from the OpenShift Pipeline Plugin will not be supported against OpenShift API Servers later than v3.11

Starting "Tag OpenShift Image" with the source [image stream:tag] "foo:45-8af086bd" from the project "bar" and destination stream(s) "foo" with tag(s) "latest" from the project "bar".

Exiting "Tag OpenShift Image" successfully.
[Pipeline] }
[Pipeline] // stage
```

**Comment by michaelsauter** [comment]
We could actually do this VERY differently going forward.

Since we want to define the OCP config via templates, and apply via Tailor during the Jenkins build, maybe we should switch the image via Tailor, too. Otherwise you might run into the situation that you deploy twice - once e.g. because you changed env variables in a deployment config, and then once because you change the image. Doing both thinks in one JSON patch would avoid that.

FYI @torstenhe 

@rattermeyer @tjaeschke @clemensutschig BTW - is the whole OCP config as part of quickstart generation / Jenkins deployment / document generation thing something that is already on your list as a pre-requisite of the release manager?

**Comment by michaelsauter** [comment]
Solved in #144.
